### PR TITLE
fix: corrected GlobalFooter shop link

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

We haven't set up the /shop redirect yet so it's a 404.

### CODE REVIEW

File: packages/gamut-labs/src/experimental/GlobalFooter/FooterNavLinks/CompanyLinks.tsx [hunk 1]
```diff
@@ -99,7 +99,7 @@ export const CompanyLinks: React.FC<CompanyLinksProps> = ({
         {userGeo !== 'IN' && (
           <FooterLinkItem>
             <Anchor
-              href="/shop"
+              href="https://codecademy.com/shop"
```
Comment (katiezutter) [inline packages/gamut-labs/src/experimental/GlobalFooter/FooterNavLinks/CompanyLinks.tsx:102]: Shouldn't it be https://shop.codecademy.com/ instead?

Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/experimental/GlobalFooter/FooterNavLinks/CompanyLinks.tsx:102]: thank you.

**Review by saghdaey — APPROVED** [review]
golden

**Review by katiezutter — APPROVED** [review]
Thank you!!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.19-alpha.166d52.0<br/>@codecademy/gamut-labs@13.4.11-alpha.166d52.0<br/>@codecademy/styleguide@33.9.11-alpha.166d52.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60775f52c2170e510153e928--gamut-preview.netlify.app>https://60775f52c2170e510153e928--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60775f52c2170e510153e928>Deploy Logs</a></p>

**Pull request merged**